pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -216,6 +217,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 		cfg.TLSOptions = nil
 	}
 
+	// Prepare cloud provider
+	cloud, err := cloudprovider.InitCloudProvider(server.CloudProvider, server.CloudConfigFile)
```

Comment:
wrap in a `if len(server.CloudProvider) > 0 {` check, otherwise you get these messages on every launch:
```
No cloud provider specified.
Successfully initialized cloud provider: "" from the config file: ""
```